ui/public/views/mobileservice.html

```diff
@@ -170,6 +170,9 @@ <h2 class="card-pf-title">
                 <uib-tab ng-if="integrations.length">
                   <uib-tab-heading>Services Integrations</uib-tab-heading>
                   <ng-include src="'extensions/mcp/views/integration-docs/mobile-service-integration.html'"></ng-include>
+                  <span ng-repeat="descLine in service.capabilities.description">
```

Comment:
This description is of the service and shouldn't appear on the integrations tab. This tab is only for integrations to the service.
Wireframe of integrations:

![integartions-wireframe](https://user-images.githubusercontent.com/22113654/31994289-018465d6-b978-11e7-8db5-2e40d5b702cd.png)